circle.yml

```diff
@@ -1,15 +1,13 @@
-dependencies:
-  pre:
-    - rm -r ~/virtualenvs
-
 machine:
   python:
-    version: 2.7
+    version: '2.7.10'
+
+dependencies:
+  override:
+    - pip install -U pip
+    - pip install -U tox tox-pyenv
+    - pyenv local 2.7.9 3.4.3 3.5.0
```

Comment:
Looks like there's a 3.5.1 out, as well as 2.7.11; should we use those as the latest? Or are these matching what CircleCI currently has?